### DISCUSSION TITLE
Fixing Table and Column's Width

### DISCUSSION THREAD

**Discussion by myhendry** [saadeghi/daisyui#532] [Q&A] [answered]
Hi. Is it possible to fix the width of the columns and if there are too many words in a table data, it will simply flow into a new line without affecting the column's width?

I'm rendering 2 tables one on top of another on a page. The width of the table and its column needs to be fixed otherwise the 2 tables will look different hence awkward

What can I do to fix the width of the table and its columns? Thanks

**Comment by saadeghi** [comment, marked as answer]
You can set `min-width` or `max-width` to each table cell:  
https://play.tailwindcss.com/ENdbcHKWmF  
You can also use `whitespace-normal` class if you want the text to wrap inside the cell.

Let me know if you have any questions

**Reply by PI-Gorbo** [reply]
I have been looking for this solution for ages. Thank you so very much

**Reply by designly1** [reply]
`whitespace-normal` did it for me
